### ISSUE TITLE
RCS1018 is not "aware" of the "file" access modifier.

### ISSUE THREAD

**Issue by amkoval** [dotnet/roslynator#1509]
**Product and Version Used**: 

"Roslynator 2022" Visual Studio extension.

**Steps to Reproduce**:

Create a class with the "file" access modifier:

```csharp
file static class SomeFileScoppedClass
{
}
```

**Actual Behavior**:

RCS1018 is highlighted. The "quick fix" is trying to add the "intermediate" access modifier in front of the "file":

```csharp
intermediate file static class SomeFileScoppedClass
{
}
```

**Expected Behavior**:
RCS1018 should not be triggered when the "file" access modifier is used.
